examples/custom-ethers-pkg/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Defender, DefenderOptions } from '@openzeppelin/defender-sdk';
+import { ActionEvent } from '@openzeppelin/defender-sdk-action-client';
+import { ContractFactory } from 'ethers';
+import ERC20Abi from '../erc20.json';
+import ERC20Bytecode from '../bytecode.json';
+
+export async function handler(event: ActionEvent & DefenderOptions) {
+  const client = new Defender(event);
+  const provider = client.relaySigner.getProvider();
+  const signer = await client.relaySigner.getSigner(provider, { speed: 'fast' });
+
+  const erc20BytesLike = ERC20Bytecode[0].data.bytecode.object;
+
+  // Deploy new ERC20 contract
+  const factory = new ContractFactory(ERC20Abi, erc20BytesLike, signer);
```

Comment:
TypeScript is getting mad at this line @MCarlomagno I'm not sure how to fix it properly.

Reply:
Issue was due to a mismatch between different module types of the ethers library being used in the project. Fix was to update the paths configuration in `tsconfig.json` to ensure all ethers imports resolve to the same module.